testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -110,14 +187,34 @@ fn upgrade_helm_release(
     release_name: String,
     helm_chart: String,
     options: &[String],
+    kube_namespace: String,
 ) -> Result<()> {
+    // only create cluster-level resources once
+    let psp_values = match kube_namespace.as_str() {
```

Comment:
i think consider not creating cluster level resources altogether - as mentioned I don't think we need PSP anyways and it just introduces a bunch of complexity both in the charts and the Rust code.